src/dra/5298C.c

```diff
@@ -3170,9 +3170,9 @@ void func_800FA3C4(s32 arg0, s32 arg1, s32 arg2) {
 
     // Here is where we use the menuitem, again, FAKE.
     if (D_801375CC.equipTypeFilter == 0) {
-        g_MenuNavigation.scrollEquipHand = menuitem[11];
+        g_MenuNavigation.scrollEquipType[HAND_TYPE] = menuitem[11];
     } else {
-        g_MenuNavigation.scrollEquipAccessories[D_801375D4] = menuitem[11];
```

Comment:
My guess is that this is split into hand and body like the equips https://github.com/Xeeynamo/sotn-decomp/blob/c443cdb1eee9053b6202cb8e60a4f3ff124a40b2/include/game.h#L775

That would explain the reason to use an if statement and the filter